src/Cake.Core/Scripting/ScriptRunner.cs

```diff
@@ -212,15 +212,15 @@ private DirectoryPath GetToolPath(DirectoryPath root)
             return root.Combine("tools");
         }
 
-        private DirectoryPath GetAddinPath(DirectoryPath applicationRoot)
+        private DirectoryPath GetAddinPath(DirectoryPath root)
         {
             var addinPath = _configuration.GetValue(Constants.Paths.Addins);
             if (!string.IsNullOrWhiteSpace(addinPath))
             {
                 return new DirectoryPath(addinPath).MakeAbsolute(_environment);
             }
 
-            return applicationRoot.Combine("../Addins").Collapse();
+            return root.Combine("tools/Addins").Collapse();
```

Comment:
@patriksvensson Well, that depends. http://cakebuild.net/docs/fundamentals/default-configuration-values clearly states that the _Addins Path_'s default is `./tools/Addins`. No relationship to the _Tools Path_ is mentioned.